ui/src/app/components/webcomponents/dry-run-form.component.ts

```diff
@@ -393,20 +395,29 @@ export class DryRunFormComponent extends BaseComponent implements OnInit {
       res => {
         res.forEach((url) => this.invalidUrls.push(url));
       },
-      err => {}
+      err => {
+      }
     )
   }
 
   disableRunButton() {
-    return this.saving || this.savingConfig  || this.emptyElements.length>0 || !this.dryExecutionForm?.valid
-      || (!this.isHybrid &&  this.invalidUrls.length>0) || (this.isHybrid && this.zeroActiveAgents) ;
+    return this.saving || this.savingConfig || this.emptyElements.length > 0 || !this.dryExecutionForm?.valid
+      || (!this.isHybrid && this.invalidUrls.length > 0) || (this.isHybrid && this.zeroActiveAgents);
   }
 
   private searchDryRunConfigs(term: any) {
     this.filteredDryRunConfigs = this.configurations.filter(config => config.name.includes(term.trim()))
   }
 
-  closeDialogTab(){
+  closeDialogTab() {
     this.dialogRef.close();
   }
+
+  navigateToResults(res) {
+    if (this.router.url.includes('test_case_results')) {
+      window.location.href = window.location.origin + buildUrl([InternalRoute.UI, InternalRoute.TD, InternalRoute.TEST_CASE_RESULTS]) + res?.content[0]?.id;
```

Comment:
why do we need to reset the location href? @manikanta-reddy-t

Reply:
when the url already contains test_case_results in the url, it is not getting refresh , to do refresh in that case we are using location href.